.github/workflows/build_release_candidates.yml

```diff
@@ -0,0 +1,39 @@
+name: Build Release Candidates
+
+concurrency:
+  group: ${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  linux:
+    name: Build Linux Release Candidate
+    runs-on: ubuntu-20.04
+    timeout-minutes: 15
+    env:
+      SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+
+      - name: Install Node
+        uses: actions/setup-node@v2.2.0
+        with:
+          node-version: 16
+
+      - name: Install NPM Dependencies
+        run: npm ci
+
+      - name: Build Linux Release Candidate
+        run: npm run action src/electron/build_linux_release
+
+      - name: Upload Linux Release Candidate
+        uses: actions/upload-artifact@v2
+        with:
+          name: release-linux-${{ github.sha }}
```

Comment:
`github.sha` gives the full sha of the commit that triggered the build - we could truncate that, but there's always the slim chance the short sha refers to more than one commit...